### PR TITLE
Fix objects merging, Fix casting BigNumber to string

### PR DESCRIPTION
**Pull request by allush**

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/symbiosis-finance/index.js exports TVL: 
        
 
 
    ethereum                  2.47 M
    avax                      4.11 M
    boba                      503.72 k
    bsc                       2.79 M
    polygon                   3.13 M
    
    total                    13.00 M

File: projects/symbiosis-finance/index.js [hunk 1]
```diff
@@ -136,21 +136,21 @@ async function tvl(chainName, timestamp, block, chainBlocks) {
     })
   })
 
-  const poolBalances = (await Promise.all(poolBalancePromises))
+  const poolBalances = await Promise.all(poolBalancePromises)
+  const allBalances = poolBalances
```
Comment (g1nt0ki) [inline projects/symbiosis-finance/index.js:140]: suggestion:  you can use `sdk.util.sumSingleBalance(balances,...` here

**Review by g1nt0ki — APPROVED** [review]

File: projects/symbiosis-finance/index.js (hunk 1 above)
Comment (allush) [reply projects/symbiosis-finance/index.js:140]: thank you very much! I followed your recommendation.

**Comment by llamatester** [comment]
The adapter at projects/symbiosis-finance/index.js exports TVL: 
        
 
 
    avax                      4.11 M
    ethereum                  2.47 M
    boba                      503.79 k
    bsc                       2.79 M
    polygon                   3.63 M
    
    total                    13.50 M

**Pull request merged**